src/components/Modal/index.tsx

```diff
@@ -39,13 +51,19 @@ export type ModalProps = {
  * @param props.title A title string
  * @param props.actions A component that if provided will add a "actions" footer
  * @param props.children The provided children content
+ * @param props.isClosable If true the close button won't be displayed and user won't be able to close modal by any means
+ * @param props.variant What style should be displayed
+ * @param props.removePadding Whether the padding of content container should be removed
  */
 export const Modal: React.FC<ModalProps> = ({
   onToggle,
   isOpen: isOpenProps,
   isHeaderAbsolute,
   title,
   actions,
+  removePadding,
+  variant = 'Default',
```

Comment:
Please replace for `variant = 'default'`. Is your TypeScript running? That assignment should have raised an error.